config.json

```diff
@@ -243,6 +243,18 @@
       "topics": [
         "Result"
       ]
+    },
+	{
+      "uuid": "10923b0b-c726-44a7-9e02-b775e7b8b237",
+      "slug": "simple-linked-list",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 4,
+      "topics": [
+		"lists",
```

Comment:
I would suggest that the indentation here use spaces instead of tabs, otherwise formatters such as https://github.com/exercism/rust/blob/master/bin/init_exercise.py will cause this line to change for a potentially unrelated change.